core/src/main/kotlin/com/malinskiy/marathon/execution/strategy/SortingStrategy.kt

```diff
@@ -1,8 +1,8 @@
 package com.malinskiy.marathon.execution.strategy
 
-import com.malinskiy.marathon.analytics.metrics.MetricsProvider
+import com.malinskiy.marathon.analytics.external.MetricsProvider
 import com.malinskiy.marathon.test.Test
-import java.util.Comparator
+import java.util.*
```

Comment:
why use a wildcard import instead of Comparator import?

Reply:
no idea why IntelliJ chose this. fixed